app/javascript/__tests__/dashboard.test.js

```diff
@@ -1,49 +0,0 @@
-/* eslint-env jest */
```

Comment:
This test was failing because the functionality it was testing (table#case_contacts) is no longer used in the app. Also, it was verifying that a scrollable DataTable was working, but the app does not use any scrolling DataTables either. So since it's no longer helping, it figured it made sense to just delete it.

Reply:
Thanks! :)